src/app/base/components/heuristic-statuses/heuristic-statuses.component.ts

```diff
@@ -14,8 +15,11 @@ export class HeuristicStatusesComponent implements OnChanges {
   dataTip: string;
   statusName: string;
   color: string;
+  isAttributesOrSettingsPage: boolean;
 
-  constructor() { }
+  constructor(private router: Router) {
+    this.isAttributesOrSettingsPage = this.router.url.indexOf('attributes') > -1 || this.router.url.indexOf('settings') > -1 ? true : false;
```

Comment:
this is decided on the router url. I think we might want to change it to an input e.g. initialCaption that is 'Initial' by default and can be overwritten. 
Depending on an URL in a component feels a bit unreliable